src/main/java/cinema/shop/model/ShoppingCart.java

```diff
@@ -0,0 +1,77 @@
+package cinema.shop.model;
+
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @OneToOne(cascade = CascadeType.MERGE)
```

Comment:
Optional - why do you use cascade here?

Reply:
I didn't correct this from the time I was using `merge()` on shopping cart. I'll remove it, THX :)